sourcegraph/sourcegraph
# cody: beta notice on site configuration

Adds Cody beta notice to site configuration when completions are enabled.

## Demo


https://user-images.githubusercontent.com/6417322/222157226-b8fd298d-2ffa-44c2-aa7e-bc32d0e39855.mov



## Test plan


* Disable or delete completions from site config
* Enable completions in site config
* You should see a notice appear above the editor containing the Cody beta information


## App preview:

- [Web](https://sg-web-rn-cody-beta-notice.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
